css/evaluations.css

```diff
@@ -0,0 +1,110 @@
+body{
+	background-color: white;
+}
+
+.logoClass{
```

Comment:
So, this is the `logoClass` class? Might I suggest renaming the `logoClass` to `logoLogoClass`, such that it becomes the `logoLogoClass` class? :P